library/constant_time_impl.h

```diff
@@ -31,7 +31,7 @@
  * Disable -Wredundant-decls so that gcc does not warn about this. This is re-enabled
  * at the bottom of this file.
  */
-#ifdef __GNUC__
+#ifdef __GNUC__ > 4
```

Comment:
This isn't valid C pre-processor syntax, which should be `# ifdef identifier new-line`

Reply:
sorry, that was a copy and paste mistake. I will fix that.

Reply:
This fix may `pop` what was not pushed.

Reply:
That would be best dealt with by something like we have in #8445

Reply:
I now fixed it by fixing the check without the pop pragma.